src/ex3/README.md

```diff
@@ -0,0 +1,33 @@
+# Exercise 3 - Node.js, CLI
+
+Time for task #3!<br>
+
+## In this section you will practice
+
+**Node.js** - Build your first node.js cli app <br>
+**Npm** - Work with external libraries <br>
+**Files** - Read and write from the file system <br>
+
+## What you are going to build
+
+We already have a beautiful todo app that will help us to catch all the pokemons. <br>
+Now let's create cli tool, so we can add, delete, and view todos like a real pro 👩‍💻 🧑‍💻
+
+Use your code from the last exercise and refactor where needed.
+
+### The requirements:
+
+- [x] Build node.js cli todo app
+- [x] Detailed help option
+- [x] Read/Write/Delete todos from a file
+
+When you finish it should look like this:
+
+![](../assets/cli.gif)
+
+### Bonus
+
+- [x] Publish your code as npm package and run it with npx
+- [x] Add colors to your cli tool
+- [x] Add Inquirer.js and make you cli tool interactive
+- [ ] Display pokemon image (ascii art)
```

Comment:
You can use an api like image-to-asci for something like this if your curious :)
[image-to-ascii - npmhttps://www.npmjs.com › package › image-to-ascii](https://www.npmjs.com/package/image-to-ascii)